recipes/libbezier/bld.bat

```diff
@@ -0,0 +1,26 @@
+:: NOTE: This assumes the following environment variables have been set.
+::       - `%SUBDIR%`
+::       - `%c_compiler%`
+::       - `%LIBRARY_PREFIX%`
+:: H/T: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html
+
+mkdir build
+cd build
+
+:: Configure.
+cmake                                              ^
+    -G "MinGW Makefiles"                           ^
```

Comment:
Should I use `-G "%CMAKE_GENERATOR%"` here? AFAIK none of the other CMake generators on Windows even support Fortran, though I am unclear on what the Intel / `ifort` story is on Windows.